web/src/component/main-shared/JuiceSlide.tsx

```diff
@@ -0,0 +1,51 @@
+import { Swiper, SwiperSlide } from "swiper/react";
+import JuiceItem from "./JuiceItem";
+import { A11y, Navigation, Pagination, Scrollbar } from "swiper/modules";
+
+interface JuiceSlideProps {
+  totalJuice: number;
+}
+
+export default function JuiceSlide(props: JuiceSlideProps) {
+  const slideNum = Math.floor(props.totalJuice / 14) + 1;
+  const SLIDE_MAP = [];
+  for (let pages = 0; pages < slideNum; pages++) {
```

Comment:
이 과정은 JS의 Array 생성자를 쓰는게 더 간결할듯
```suggestion
const SLIDE_MAP = [...new Array(slideNum)]
```

Reply:
### 수정 코드
```tsx
const SLIDE_MAP = [...new Array(slideNum)].map((_, index) => index + 1);
```
### 1. Array() 생성자
- 새로운 Array 객체를 생성할 때 사용한다.
- `new Array(2)` : 2개의 빈 공간을 가진 배열(값이 없는 상태)

### 2. Array() 생성자 + 전개 연산자
- 배열을 하나씩 펼쳐서 리턴해서 새로운 배열을 만든다.
- `...new Array(2)` => `[undefined, undefined]`

### 3. map()
- 기본 형태 : `map((value, index, array) => ())`
- `map()`을 이용해서 undefined 값을 slideNum의 값으로 변경한다.
- 배열의 요소를 나타내는 `value`는 `SLIDE_MAP`에서는 `undefined`이기 때문에 `_`로 표시한다. (=> value라고 써도 상관없지만, 관습적으로 `undefined`이거나 사용하지 않을 경우에 `_`을 쓴다고 한다. )